platform/admin/core/admin-core-impl/src/test/java/org/codice/ddf/admin/core/impl/AdminConsoleServiceTest.java

```diff
@@ -810,6 +810,22 @@ public void testUpdate() throws Exception {
     assertThat(((Integer[]) captor.getValue().get(arrayInteger)).length, equalTo(0));
   }
 
+  @Test(expected = IllegalArgumentException.class)
+  public void testSanitizeUIConfiguration() throws Exception {
+    AdminConsoleService configAdmin = getConfigAdmin();
+
+    // Initialize illegal values.
+    Map<String, Object> currentProps = new Hashtable<>();
+    currentProps.put("color", "yellow;");
+    currentProps.put(
+        "background",
+        "black; float: left; text-align: center; width: 120px; border: 1px solid gray; margin: 4px; padding: 6px;");
+
+    ArgumentCaptor<Dictionary> captor = ArgumentCaptor.forClass(Dictionary.class);
+    configAdmin.update(UI_CONFIG_PID, currentProps);
+    verify(testConfig, times(1)).update(captor.capture());
```

Comment:
❗️ You've created an argument captor but you're not using it to validate the captured value. Instead replace the call to `capture()` with a predefined predicate to put a bounds on expected arguments, for example:
```
verify(testConfig, times(1)).update(any(Dictionary.class));
```